lib/stack_master/template_compilers/cfndsl.rb

```diff
@@ -4,10 +4,15 @@ def self.require_dependencies
       require 'cfndsl'
     end
 
-    def self.compile(template_file_path)
-      ::CfnDsl.eval_file_with_extras(template_file_path).to_json
+    def self.compile(template_file_path, compiler_options = {})
+      if compiler_options["disable_binding"]
+        ::CfnDsl.disable_binding
+      end
```

Comment:
Could we reset this option after compiling? It's possible to run multiple stack updates in the same process `stack_master apply production` and this config would leak to subsequent applies.